cheatsheets/Access_Control_Cheat_Sheet.md

```diff
@@ -45,22 +45,22 @@ The areas of caution while using RBAC are:
 
 # Discretionary Access Control (DAC)
 
-Discretionary Access Control (DAC) is a means of restricting access to information based on the identity of users and/or membership in certain groups. Access decisions are typically based on the authorizations granted to a user based on the credentials he presented at the time of authentication (user name, password, hardware/software token, etc.). In most typical DAC models, the owner of information or any resource is able to change its permissions at his discretion (thus the name).
+Discretionary Access Control (DAC) is a means of restricting access to information based on the identity of users and/or membership in certain groups. Access decisions are typically based on the authorizations granted to a user based on the credentials he presented at the time of authentication (user name, password, hardware/software token, etc.). In most typical DAC models, the owner of the information or any resource is able to change its permissions at his discretion (thus the name).
 
-A DAC framework can provide web application security administrators with the ability to implement fine grained access control. This model can be a basis for data based access control implementation
+A DAC framework can provide web application security administrators with the ability to implement fine-grained access control. This model can be a basis for data-based access control implementation
 
 The advantages of using this model are:
 
 - Easy to use
 - Easy to administer
-- Aligns to the principle of least privileges.
+- It aligns to the principle of least privileges.
```

Comment:
This is not inconsistent with the other elements of the list. They should either all be as they are (Easy, Aligns, Object), or all like this (It is easy, it aligns, the object has..)